### ISSUE TITLE
issues with blame mode from magit

### ISSUE THREAD

**Issue by emmanueltouzery** [alpaker/fill-column-indicator#59]
I have noticed some rendering artifacts of the fill-column-indicator when combined with magit blame mode, see pictures.

![screenshot from 2015-06-01 23-18-40](https://cloud.githubusercontent.com/assets/339433/7923600/b9c535ca-08b4-11e5-98ed-18752d83972b.png)
![screenshot from 2015-06-01 23-18-44](https://cloud.githubusercontent.com/assets/339433/7923599/b9c4e17e-08b4-11e5-9f1e-623d90d880c8.png)

**Comment by alpaker** [comment]
Thanks for the report.  I haven't had a chance to look into yet, but this
is a familiar sort of compatibility issue with other modes that do things
at the start and/or end of lines.  I'll dig in when I can.  Sorry I can't
be of more help at the moment.

**Comment by tvaughan** [comment]
I ran into some really weird problems with fci and the magit status buffer in v2.2.x. This solves the symptom not the problem, but it works well enough for me. YMMV.

```lisp
(define-globalized-minor-mode global-fci-mode fci-mode
  (lambda ()
    (unless buffer-read-only
      (fci-mode t))))

(global-fci-mode t)
```

**Comment by alpaker** [comment]
Just FYI, I'm abandoning this package and welcoming requests to be take over as maintainer.

**Comment by alpaker** [comment]
The functionality provided by this package has now been implemented natively in Emacs as `display-fill-column-indicator-mode`, available as of version 27.0.90. Unless you are forced to use an older Emacs, you should use the native implementation instead of `fci-mode`. The former is unaffected by most of the compatibility issues that this add-on has.
